documentation/Design-Guidelines.md

```diff
@@ -41,11 +41,12 @@
 
 # Libertinus design guidelines
 
-These are short guidelines for all volunteers who love libre fonts in general, the Libertinus font family especially, and who think about contributing to the latter.
-Fonts contain dozens of glyphs. Libertinus for example has several thousands of glyphs, and they all fit together while being different. The glyphs have to fit together - in the aspect of design, their optical weight, and their spacing. Some technical issues have to be met to achive that.
-Sounds complicated? Nevermind, we will get you into it. This is what these guidelines are for.
+These are short guidelines for all volunteers who love libre fonts in general, the Libertinus font family in specific, and who are considering contributing to the latter.
```

Comment:
This is just a stylistic choice, but ["in general / in particular"](https://www.google.com/search?q=%22in+general%22+and+%22in+particular%22) seems to be more common than ["in general / in specific"](https://www.google.com/search?q=%22in+general%22+and+%22in+particular%22) (by about an order of magnitude, according to the linked Google search results). Both are valid, but using a more common expression can make this passage slightly more approachable.

Reply:
Fair enough. I'm accustomed to "in general", but either way is better than "...especially" — which would have worked with other wording at the beginning but the whole thing was just a miss-match.

Along with fixing the word you brought up I did a little more work moving through that file, but frankly it needs a _lot_ more help!